jdi-light/src/main/java/com/epam/jdi/light/elements/pageobjects/annotations/WebAnnotationsUtil.java

```diff
@@ -154,5 +167,24 @@ public static By findByToBy(WithText locator){
         if (locator == null) return null;
         return withText(locator.value());
     }
+    private static By getHowLocator(org.openqa.selenium.support.FindBy locator) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'getHowLocator(org.openqa.selenium.support.FindBy)' has an NPath complexity of 256](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646844)